src/components/pre-playback-play-overlay/pre-playback-play-overlay.js

```diff
@@ -126,6 +114,7 @@ class PrePlaybackPlayOverlay extends BaseComponent {
     if ((!props.isEnded && !props.prePlayback) || (!props.isEnded && this.autoplay)) {
       return undefined;
     }
+    let playerIsReady = this.player.config.playback.preload === "auto" ? this.state.isPlayerReady : true;
```

Comment:
use const
why state name is isPlayerReady and var name is  playerIsReady?

Reply:
changed to const + naming, thanks